marklogic-data-hub/src/test/ml-modules/root/test/suites/data-hub/5/entities/search/addPropertiesToSearchResponse.sjs

```diff
@@ -480,17 +480,17 @@ function verifyPropertiesForAllEntitiesOption() {
     ]
   };
   const entityName = null; //Indicates that user has chosen 'All Entities' option
-  
+
```

Comment:
This test is for All Entities. Add a test when an Entity is selected and verify if the  entityName and createdOn are returned